pydevd_concurrency_analyser/pydevd_thread_wrappers.py

```diff
@@ -0,0 +1,87 @@
+
+import _pydev_threading as threading
+
+
+def wrapper(fun):
+    def pydev_after_run_call():
+        pass
+
+    def inner(*args, **kwargs):
+        fun(*args, **kwargs)
+        pydev_after_run_call()
+    return inner
+
+
+def wrap_attr(obj, attr):
+    t_save_start = getattr(obj, attr)
+    setattr(obj, attr, wrapper(t_save_start))
+    setattr(obj, "_pydev_run_patched", True)
+
+
+class ObjectWrapper(object):
+    def __init__(self, object):
```

Comment:
I think it'd be nice renaming the parameter from object -> obj so that it doesn't conflict with the object builtin.